### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by benihime1458**

### CODE REVIEW

File: client/src/components/App.jsx [hunk 1]
```diff
@@ -27,6 +27,9 @@ class App extends React.Component {
   render () {
     return (
       <div>
+        <div>
+          <p> <i class="fas fa-comment-alt"></i> {this.state.comments.length} comments </p>
```
Comment (kristie-zheng) [inline client/src/components/App.jsx]: Style guide requests that empty tags are self closing, like
<i class="..." />

File: client/src/components/CommentEntry.jsx [hunk 2]
```diff
@@ -1,48 +1,69 @@
...
+    }
+
+    return result;
+  }
+
   return (
     <CommentContainer>
-      <Space></Space> 
-      <Avatar></Avatar> 
+      <Avatar><i class="fas fa-user-circle fa-2x"></i></Avatar>
```
Comment (kristie-zheng) [inline client/src/components/CommentEntry.jsx]: See above + have open/close tags on separate lines

File: client/src/components/App.jsx (hunk 1 above)
Comment (benihime1458) [reply client/src/components/App.jsx]: Okie doke.

File: client/src/components/CommentEntry.jsx (hunk 2 above)
Comment (benihime1458) [reply client/src/components/CommentEntry.jsx]: Yee.

File: client/src/components/CommentsList.jsx [hunk 3]
```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import CommentEntry from './CommentEntry';
 
 const CommentsList = props => (
-  <div className='comments'>
+  <div>
     {props.commentsList.map(comment => <CommentEntry key={comment.id} comment={comment} />
```
[the remark this answers is not in the document]
Comment (benihime1458) [reply client/src/components/CommentsList.jsx]: Gotcha!

File: db/commentGen.js [hunk 4]
```diff
@@ -124,6 +124,10 @@ const commentGen = (hipString) => {
       message += `${hipArray[Math.floor(Math.random() * hipArray.length)]} `;
     }
     comment.text = message;
+    
+    // songtime, timestamp
+    comment.songtime = Math.round(Math.random() * 600);
```
[the remark this answers is not in the document]
Comment (benihime1458) [reply db/commentGen.js]: Thanks!

**Pull request merged**